cmake/PackageConfig.cmake.in

```diff
@@ -0,0 +1,12 @@
+@PACKAGE_INIT@
+
+#  * @PROJECT_NAME@::@PROJECT_NAME@_4 - real32 library target
+#  * @PROJECT_NAME@::@PROJECT_NAME@_8 - real64 library target
+
+# Include targets file.  This will create IMPORTED target @PROJECT_NAME@
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
+
```

Comment:
we are adding a dependency on OpenMP, so that should be found if this library was compiled with that option.
Something like this:
```
# ON/OFF implies ip was compiled with/without OpenMP
if(@OpenMP_Fortran_FOUND@)
  find_dependency(OpenMP)
endif()
```
The documentation is [here](https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html).
It says there is no need to pass arguments such as `REQUIRED` or `COMPONENTS Fortran` as they are passed from the original call to `find_package`, but we should check that is correctly done.